pkg/image/manifest.go

```diff
@@ -84,8 +182,9 @@ func NewRegistryList(repoConfig, k8sVersion string) (*RegistryList, error) {
 		PrivateRegistry:         privateRegistry,
 		QuayIncubator:           quayIncubator,
 		QuayK8sCSI:              quayK8sCSI,
-		SampleRegistry:          sampleRegistry,
 		PromoterE2eRegistry:     promoterE2eRegistry,
+		SampleRegistry:          sampleRegistry,
```

Comment:
I haven't checked all the registries, but this one looks like it's no longer being used so you could remove it.

Reply:
👍